.code-samples.meilisearch.yaml

```diff
@@ -450,3 +450,58 @@ tenant_token_guide_search_sdk_1: |-
   front_end_client = MeiliSearch::Client.new('http://127.0.0.1:7700', token)
 
   front_end_client.index('patient_medical_records').search('blood test')
+getting_started_update_rankingRules: |-
+  client.index('movies').update_ranking_rules([
+    'exactness',
+    'words',
+    'typo',
+    'proximity',
+    'attribute',
+    'sort',
+    'release_date:asc',
+    'rank:desc'
+  ])
+getting_started_update_searchableAttributes: |-
+  client.index('movies').update_searchable_attributes([
+    'title'
+  ])
+getting_started_update_stop_words: |-
+  client.index('movies').update_searchable_attributes([
+    'title'
+  ])
+getting_started_check_task_status_md: |-
+  ```rb
+  client.index('movies').task(0)
+  ```
+getting_started_synonyms: |-
+  client.index('movies').update_synonyms({
+    winnie: ['piglet'],
+    piglet: ['winnie']
+  })
+getting_started_update_displayedAttributes: |-
+  client.index('movies').update_displayed_attributes([
+    'title',
+    'overview',
+    'poster'
+  ])
+getting_started_communicating_with_a_protected_instance: |-
+  client = MeiliSearch::Client.new('http://127.0.0.1:7700', 'apiKey')
+  client.index('movies').search
+getting_started_add_meteorites: |-
+ should we add a file.read ?
```

Comment:
I take the example from the `getting_started_add_documents_md` code sample [in python I write it like]
(https://github.com/meilisearch/meilisearch-python/blob/6722b031e540e73ae776a927f74eea7a38076344/.code-samples.meilisearch.yaml#L369)

Reply:
Yes, this is something I care about because it could be hard to do that on some packages (like, Swift and Dart) 🤔

Reply:
@bidoubiwa what do you think about this?

Reply:
In the other examples I imported the JSON file so I think we should continue doing that, but in languages where this is to complex we could adapt.